simple/stats/importer.py

```diff
@@ -98,6 +100,25 @@ def _rename_columns(self) -> None:
 
     self.df = self.df.rename(columns=renamed)
 
+  def _add_entity_nodes(self) -> None:
+    if not self.entity_type:
+      self.entity_type = self._resolve_entity_type()
+      if self.entity_type:
+        logging.info("Resolved entity type: %s", self.entity_type)
+    if not self.entity_type:
+      logging.warning(
+          "Could not resolve entity type. Entity triples will not be imported.")
+      return
+    self.nodes.entities_with_type(self.df.iloc[:, 0].tolist(), self.entity_type)
+
+  def _resolve_entity_type(self) -> str:
+    all_entity_dcids = self.df.iloc[:, 0].tolist()
+    sample_entity_dcids = random.sample(all_entity_dcids,
+                                        min(len(all_entity_dcids), 5))
```

Comment:
nit: make 5 a constant; And worth documenting the behavior, ie, we assume the entity column has same type of entities.

Reply:
Done.